app/templates/confirm.njk

```diff
@@ -12,15 +13,29 @@
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds govuk-body">
+      <h1 class="govuk-heading-l">Confirm and send</h1>
+<p>I confirm that, to the best of my knowledge, the details I have provided are correct.</p>
+<p>I understand the score was based on the answers I provided.</p>
+<p>I am aware the information I submit will be checked.</p>
+<p>So that we can continue to improve our services and schemes, we may wish to contact you in the future. Please confirm if you are happy for us, or a third party working for us, to contact you.</p>
+</div>
+  <div class="govuk-grid-row govuk-body">
     <div class="govuk-grid-column-two-thirds">
       <form method="POST" action="/confirm" novalidate="novalidate">
-        <input type="hidden" name="crumb" value="{{crumb}}" />
-        {{ govukInput(inputConfirm) }}
+        {{ crumbToken(crumb)}}
+        {{ govukCheckboxes(checkboxConfirm) }}
         {{ govukButton({
-          text: "Continue"
+          text: "Confirm and send"
         }) }}
       </form>
     </div>
   </div>
+    <div class="govuk-grid-row govuk-body">
+    <div class="govuk-grid-column-two-thirds">
+  <p>The Department for Environment, Food and Rural Affairs (Defra) is the data controller for personal data you give to RPA. For information on how we handle personal data go to GOV.UK and search</p>
```

Comment:
Check this content. The sentence `For information on how we handle personal data go to GOV.UK and search` doesn't look right, and you have the link below

Reply:
https://capgemini.invisionapp.com/share/YZ10M5WAWD92#/screens/448577412
just followed this.